ci/do_ci.sh

```diff
@@ -459,6 +459,10 @@ EOF
   make load_plugin_example
   examples/plugin/load/load_plugin_example ${PLUGIN_DIR}/libexample_plugin.so /dev/null
   exit 0
+elif [[ "$1" == "bazel.no_bzlmod.test" ]]; then
+  bazel $BAZEL_STARTUP_OPTIONS build --enable_bzlmod=false $BAZEL_OPTIONS //...
+  bazel $BAZEL_STARTUP_OPTIONS test --enable_bzlmod=false $BAZEL_TEST_OPTIONS //...
```

Comment:
So, does this build ignores the content of file MODULE.bazel ?

Should we have another build with bzlmod enabled ?

Reply:
bazel 7.x enables bzlmod by default, so when this merges all bazel jobs that already exist will use this new method by default. so I added this one to make sure the old way continues to work at least for this single build, and figure we can run all the other CI jobs with the new way